staramr/blast/BlastHandler.py

```diff
@@ -90,7 +95,11 @@ def run_blasts(self, files) -> None:
         db_files = self._make_db_from_input_files(self._input_genomes_tmp_dir, files)
         logger.debug("Done making blast databases for input files")
 
+        logger.info("Scheduling MLST for input files")
```

Comment:
I think, since we are now handling both **blast** and **mlst** jobs in this class, that it should be renamed from **BlastHandler.py** to **JobHandler.py**. Could you make that change here?

Reply:
Yeah, changed it in commit e4f2216

Reply:
Thanks.